### PR TITLE
JIRA ID : PD-896

### PR DESCRIPTION
**Pull request by ananthnpai**
DESCRIPTION :SMB GA ver 3.7003 config jsons without release checksum
TEST RUN :
COMMENTS :

### CODE REVIEW

**Review by anupnayakCLS — CHANGES_REQUESTED** [review]

File: SMB/Configurations/clcommon__Execution_Criteria__c/Max_Combined_Limit__c.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (ananthnpai) [reply SMB/Configurations/clcommon__Execution_Criteria__c/Max_Combined_Limit__c.json:1]: yes, required records were compared based on GA ver v_3.7003 CSV files present in repo https://github.com/cloudlending/NA-Banking-Solutions/tree/v_3.7003_Tag/src/main/default/documents/Configuration_Data

File: SMB/Configurations/genesis__Policy_Exception_Definition__c/Beneficial_Owner_Name_Address_Verification_KYC.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
      "genesis__Policy_Condition_Name__c": "Beneficial Owner Name/Address Verification (KYC)",
      "genesis__Portal_Description__c": "Any one of the following documents can be used to verify the name and address of each beneficial owner.",
      "genesis__Portal_Title__c": "Name/Address Verification (Beneficial Owner)",
+     "genesis__Product_Department_Task_Association__r": null,
```
[the remark this answers is not in the document]
Comment (ananthnpai) [reply SMB/Configurations/genesis__Policy_Exception_Definition__c/Beneficial_Owner_Name_Address_Verification_KYC.json:22]: standard config sequence order was updated for "genesis__Policy_Exception_Definition__c"

File: SMB/Configurations/standard_configuration.json [hunk 3]
```diff
@@ -1,260 +1,248 @@
...
-    },
-    {
-         "Sequence": 30,
-         "Object API Name": "intframework__Integration_API_Configuration__c",
-         "External Id Field": "intframework__External_Id__c",
-         "label": "Integration API Configuration"
-    },
-    {
-        "Sequence": 31,
-        "Object API Name": "ints__Credit_Pull_Options__c",
```
[the remark this answers is not in the document]
Comment (ananthnpai) [reply SMB/Configurations/standard_configuration.json:184]: yes, as per discussion with Paras these are not required

File: SMB/Configurations/standard_configuration.json [hunk 4]
```diff
@@ -1,260 +1,248 @@
...
-        "External Id Field": "ints__External_Id__c",
-        "label": "Meridian Link Credit Pull Options"
-    },
-    {
-        "Sequence": 32,
-        "Object API Name": "ints__Credit_Pull_Parameters__c",
```
[the remark this answers is not in the document]
Comment (ananthnpai) [reply SMB/Configurations/standard_configuration.json:190]: yes, as per discussion with @paras-dev  these are not required

File: SMB/Configurations/standard_configuration.json [hunk 5]
```diff
@@ -1,260 +1,248 @@
...
+          "label": "Product Department Junction"
+     },
+     {
+          "Sequence": 44,
+          "Object API Name": "genesis__ProductDepartment_Task_Junction__c",
+          "External Id Field": "genesis__External_ID__c",
+          "label": "Product Department Task Junction"
+     },
+	 {
+          "Sequence": 45,
```
[the remark this answers is not in the document]
Comment (ananthnpai) [reply SMB/Configurations/standard_configuration.json:123]: yes the sequence order for SMB was verified with @paras-dev and updated accordingly

**Review by anupnayakCLS — APPROVED** [review]

**Review by anupnayakCLS — APPROVED** [review]

**Review by anupnayakCLS — APPROVED** [review]

**Pull request merged**